buildsystem/dependencies.gradle

```diff
@@ -26,29 +26,29 @@ ext {
 
 
     generalDependencies = [
-            rxAndroid : "io.reactivex.rxjava2:rxandroid:${rxAndroidVersion}",
-            rxJava    : "io.reactivex.rxjava2:rxjava:${rxJavaVersion}",
-            appSupport: "com.android.support:appcompat-v7:${appSupportVersion}",
-            kotlin    : "org.jetbrains.kotlin:kotlin-stdlib:${kotlin_version}"
+            rxAndroid : "io.reactivex.rxjava2:rxandroid:$rxAndroidVersion",
+            rxJava    : "io.reactivex.rxjava2:rxjava:$rxJavaVersion",
+            appSupport: 'androidx.appcompat:appcompat:1.0.0',
```

Comment:
1-. please match the name with the dependency 
2-. make a variable for the version please

Reply:
done